src/Api.ts

```diff
@@ -1,31 +1,19 @@
 import { EmbalmerFacet__factory } from '@sarcophagus-org/sarcophagus-v2-contracts';
-import { BigNumber, ethers } from 'ethers';
-import { SarcoClient } from './SarcoClient';
+import { ethers } from 'ethers';
 import { safeContractCall } from './helpers/safeContractCall';
 import { CallOptions } from './types';
-import { goerliNetworkConfig } from './networkConfig';
 import {
   SarcophagusSettings,
   sarcophagusSettingsSchema,
   archaeologistSettingsArraySchema,
   ArchaeologistSettings,
 } from './helpers/validation';
 
-// Temporary
-// TODO: Get this from the contracts package
-const goerliDiamondAddress = '0x6B84f17bbfCe26776fEFDf5cF039cA0E66C46Caf';
-
 export class Api {
-  sarcoClient: SarcoClient;
-  embalmerFacet: ethers.Contract;
+  private embalmerFacet: ethers.Contract;
 
-  constructor(sarcoClient: SarcoClient) {
-    this.sarcoClient = sarcoClient;
-    this.embalmerFacet = new ethers.Contract(
-      goerliNetworkConfig.diamondDeployAddress,
-      EmbalmerFacet__factory.abi,
-      this.sarcoClient.signer
-    );
+  constructor(diamondDeployAddress: string, signer: ethers.Signer) {
```

Comment:
These constructors have been updated to accept only the parameters they'll need internally, since some `SarcoClient` methods have been made `private` and will not be accessible from here